docs/internships/gsod/ideas.md

```diff
@@ -142,15 +165,6 @@ The Talawa API has a number of GraphQL queries which are undocumented and have b
 * **Repos to update:** Talawa-API
 * **Skills Required:** The ideal candidate should be familiar with Node.js; GraphQL is not a strict requirement, but they must be willing to learn. Additionally, they need to be comfortable with using Git/Github. Experience in Illustration or graphic design is a plus.
 * **How we measure sucess:** Visualisations present through the base documentation and any new documentation generated from this program.
-* **Possible Mentors:** Shannika Jackson, Dominic Mills, Michael Lue, Tasneem Koushar 
+* **Possible Mentors:** Eva Sharma 
 * **Contact details:** Send your CV along with at least two technical writing samples to ![img](/img/email/mentors.png)
 
-### Revamp and Restructure the Current Documentation Page
-
-The website that currently hosts the documentation is rather simplistic and has not evolved from when the Talawa application was more refined than it is now. It no longer takes into consideration the needs of contributing to a complex, multi-layered application, and as such it needs to be remodelled and redesigned to account for a number of changes that have and are currently taking place. Remodelling involves re-writing much of the documentation already in place on the site, as well as, working with a candidate that may be adding to the documentation and ensuring that templates are in place such that the look and feel of the site is cogent and consistent. Note: it is possible to work on this task in conjunction with another one of the aforementioned tasks here..
-
-* **Repos to update:** Talawa-API
-* **Skills Required:** HTML, Javascript, CSS. Any design skills would be a plus. Must also be comfortable using Github to submit changes.
-* **How we measure sucess:** A website structured in a completely different manner, which is more efficient and effective in providing the information for a user wishing contribute to Talawa. This can be measured by the increase in new contributors via pull requests.
-* **Possible Mentors:** Shannika Jackson, Dominic Mills, Michael Lue, Tasneem Koushar 
-* **Contact details:** Send your CV along with at least two technical writing samples to ![img](/img/email/mentors.png)
```

Comment:
could you add one more project section, for documentation of Talawa-admin.

Reply:
Hey Tasneem,
I have added the additional features you asked for.
For adding another project section for documentation of Talawa-admin, I just mentioned it in the first project. Let me know if I should create another project.